lib/services/service_brokers/v2/client.rb

```diff
@@ -327,6 +327,14 @@ def fetch_service_binding(service_binding)
 
     private
 
+    def binding_required_parameters(binding)
```

Comment:
I had to pull this out into a method cause Rubocop started complaining about cyclomatic complexity. 😞